x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -193,6 +193,8 @@ static RoleDescriptor kibanaSystem(String name) {
                     .build(),
                 // Fleet telemetry queries Agent Logs indices in kibana task runner
                 RoleDescriptor.IndicesPrivileges.builder().indices("logs-elastic_agent*").privileges("read").build(),
+                // Fleet publishes Agent metrics in kibana task runner
+                RoleDescriptor.IndicesPrivileges.builder().indices("metrics-fleet_server*").privileges("auto_configure", "write").build(),
```

Comment:
`write` to allow creating documents
`auto_configure` to allow creation of index when creating documents if doesn't exist